src/styles/blocks/nav.scss

```
@@ -0,0 +1,54 @@
+.nav {
+  margin-right: 50px;
+
+  &__link {
+    display: flex;
+    color: #060b35;
+    font-size: 12px;
+    font-style: normal;
+    font-weight: 500;
+    text-transform: uppercase;
+    text-decoration: none;
+    margin: 0 10px;
+    padding: 23px 0;
+
+    &:hover {
+      color: #00acdc;
+      cursor: pointer;
+      transition: all 0.3s ease;
+    }
+  }
+
+  &__items {
+    display: flex;
+    list-style-type: none;
+    margin: 0;
+    padding: 0;
+
+    &-item {
+      margin: 0;
+      padding: 0;
+    }
+  }
+}
+
+.is-active {
+  color: #00acdc;
+  margin-left: 0;
+  padding-left: 0;
+  position: relative;
+
+  &:after {
+    content: "";
+    position: absolute;
+    background-color: #00acdc;
+    height: 4px;
+    width: 37px;
```

```suggestion
    width: 100%;
```
Don't limit the width. It should take the whole width of the relative element. In that case, it will correctly even with other links